src/commands/labs/disable.test.js

```diff
@@ -2,6 +2,12 @@
 
 import LabsDisable from './disable'
 import nock from 'nock'
+const cli = require('heroku-cli-util')
+
+// flow$ignore
+process.stdout.columns = 80 // Set screen width for consistent wrapping
+// flow$ignore
+process.stderr.columns = 80 // Set screen width for consistent wrapping
```

Comment:
can you put this into a `test/init.js` file? (or equivalent)? this is happening globally, but it would be good to do it before all the commands rather than just the ones that run before this test